product/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls   import path
+from product.views import ProductView, ReviewView, CategoryView
+
+urlpatterns = [path('product/<int:id>', ProductView.as_view()),
```

Comment:
이 뷰는 product_detail_view일까요? 
```suggestion
urlpatterns = [path('/detail/<int:product_id>', ProductView.as_view()),
```
id 라는 변수보다는 명확히 알수 있게 product_id로 주세요.